Classes/ToxWrapper/OCTTox.m

```diff
@@ -1039,3 +1055,18 @@ void avatarDataCallback(Tox *cTox,
     }
 }
 
+void fileSendRequestCallback(Tox *cTox, int32_t friendNumber, uint8_t filenumber, uint64_t filesize, const uint8_t *filename, uint16_t filename_length, void *userData)
```

Comment:
It would be better to use camelCase for variable names to have one style everythere (`fileNumber`, `fileSize`, `fileName`, `fileNameLength`).